underfs/pom.xml

```diff
@@ -51,8 +51,13 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
```

Comment:
Do we need both of these?

I would look into first upgrading slf4j to the latest, and then utilize the same log4j version in the slf4j log4j impl

Reply:
I _think_ we'll need to replace the slf4j-log4j12 with the newer log4j-slf4j-impl https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl/2.13.3

Reply:
Done